aci/data_source_aci_l3extlifp.go

```diff
@@ -32,73 +31,54 @@ func dataSourceAciLogicalInterfaceProfile() *schema.Resource {
 
 			"name_alias": &schema.Schema{
 				Type:     schema.TypeString,
-				Optional: true,
 				Computed: true,
 			},
 
 			"prio": &schema.Schema{
 				Type:     schema.TypeString,
-				Optional: true,
 				Computed: true,
 			},
 
 			"tag": &schema.Schema{
 				Type:     schema.TypeString,
-				Optional: true,
 				Computed: true,
 			},
 			"relation_l3ext_rs_l_if_p_to_netflow_monitor_pol": &schema.Schema{
 				Type:     schema.TypeSet,
-				Optional: true,
 				Computed: true,
-				Elem: &schema.Resource{
-					Schema: map[string]*schema.Schema{
-						"tn_netflow_monitor_pol_name": {
-							Type:       schema.TypeString,
-							Optional:   true,
-							Deprecated: "Use tn_netflow_monitor_pol_dn instead",
-						},
-						"tn_netflow_monitor_pol_dn": {
-							Type:     schema.TypeString,
-							Optional: true,
-						},
-						"flt_type": {
-							Type:     schema.TypeString,
-							Required: true,
-							ValidateFunc: validation.StringInSlice([]string{
-								"ipv4",
-								"ipv6",
-								"ce",
-							}, false),
-						},
-					},
-				},
 			},
```

Comment:
Elems type should be mentioned in the data source.

```
			"relation_l3ext_rs_l_if_p_to_netflow_monitor_pol": &schema.Schema{
				Type:     schema.TypeSet,
				Computed: true,
				Elem: &schema.Resource{
					Schema: map[string]*schema.Schema{
						"tn_netflow_monitor_pol_dn": {
							Type:     schema.TypeString,
							Computed: true,
						},
						"flt_type": {
							Type:     schema.TypeString,
							Computed: true,
						},
					},
				},
			},
```

Reply:
ok